python_app/app.py

```
@@ -0,0 +1,64 @@
+    return jsonify({"state": state.value})
+
+
+@app.route("/api/parse", methods=["POST"])
+def api_parse() -> Any:
+    raw: Any = request.get_json()
+    parsed: Any = parse_sensor_payload(raw)
+    return jsonify(parsed)
+
+
+if __name__ == "__main__":
+    app.run(debug=True, port=3000)
```

Ruff (0.15.6)

[error] Use of `debug=True` in Flask app detected

(S201)